pmoves/docs/AGENTS/HARDWARE_TTS_REQUIREMENTS.md

```
@@ -270,6 +280,261 @@ It creates a self-improving loop. We even run a full smoke-test harness--make ve
+| `f5_speed` | 0.5–2.0 | 1.0 | Speech speed multiplier |
+| `f5_cross_fade` | 0.0–1.0 | 0.15 | Cross-fade between chunks |
+| `f5_remove_silence` | bool | false | Strip silence from output |
+
+**Models:** F5-TTS Base, F5-TTS v1 Base, F5-TTS French, F5-TTS German, F5-TTS Japanese, F5-TTS Spanish
+
+**Multi-Language Support:** English, French, German, Japanese, Spanish (via model selection)
+
+**Reference Audio:** Required. 10-15s, clear, WAV/MP3 44.1kHz. Matching reference text improves quality.
+
+**Example — Beat-Synced Narration:**
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)